test/api/v3/unit/libs/payments.test.js

```diff
@@ -689,14 +691,51 @@ describe('payments/index', () => {
 
       it('sends a message from purchaser to recipient', async () => {
         await api.buyGems(data);
+        let msg = '\`Hello recipient, sender has sent you 4 gems!\`';
 
-        expect(user.sendMessage).to.be.calledWith(recipient, '\`Hello recipient, sender has sent you 4 gems!\`');
+        expect(user.sendMessage).to.be.calledWith(recipient, { receiverMsg: msg, senderMsg: msg });
       });
 
       it('sends a push notification if user did not gift to self', async () => {
         await api.buyGems(data);
         expect(notifications.sendNotification).to.be.calledOnce;
       });
+
+      it('sends gem donation message in each participant\'s language', async () => {
+        await recipient.update({
+          'preferences.language': 'es',
+        });
+        await user.update({
+          'preferences.language': 'cs',
+        });
+        await api.buyGems(data);
+
+        let updatedRecipient = await recipient.get('/user');
```

Comment:
Ah, sorry. I mixed it up myself. 

Just use the tests above for examples. So, you can use this expectation instead of fetching the users again: `expect(user.sendMessage).to.be.calledWith(recipient, { receiverMsg: msg, senderMsg: msg });` You already have separate tests for sendMessage. And it looks like this method is already stubbed for you.